rs/src/management/http_client.rs

```diff
@@ -56,7 +63,12 @@ impl TunnelManagementClient {
         url.query_pairs_mut().append_pair("global", "true");
 
         let request = self.make_tunnel_request(Method::GET, url, options).await?;
-        self.execute_json("list_all_tunnels", request).await
+        let response: TunnelListByRegionResponse = self.execute_json("list_all_tunnels", request).await?;
+        let mut tunnels = Vec::new();
+        for region in response.value {
+            tunnels.extend(region.value);
+        }
+        Ok(tunnels)
```

Comment:
Should be doable with something like

```suggestion
        Ok(response.value.into_iter().flat_map(|v| v.value).collect())
```

here and below